app/views/hikes/index.html.erb

```diff
@@ -13,6 +13,8 @@
                 <form method="post">
                   <div data-role="rangeslider">
                     <label for="price-min">400</label>
+                    <input type="range" name="price-min" id="price-min" value="200" min="400" max="800">
```

Comment:
Hi @EmmaSelvig I noticed that you haved duplicated the inputs for min price (`id="price-min"`).

The filter for max price is not working, I believe you just need to have 2 inputs with different names/ids (min/max) and make sure we use them when we do the search